src/nitro/communication/messages/incoming/IncomingHeader.ts

```diff
@@ -423,6 +423,7 @@ export class IncomingHeader
     public static ROOM_SETTINGS_ERROR = 2897;
     public static SHOW_ENFORCE_ROOM_CATEGORY = 3896;
     public static CUSTOM_USER_NOTIFICATION = 909;
+    public static CUSTOM_STACKING_HEIGHT_UPDATE = 0x0B00;
```

Comment:
this one seems to be a duplicate of an existing one. `0x0B00` is just hex for `2816`